cypress/e2e/po/pages/cluster-manager/machine-deployments.po.ts

```diff
@@ -0,0 +1,44 @@
+import PagePo from '@/cypress/e2e/po/pages/page.po';
+import CodeMirrorPo from '@/cypress/e2e/po/components/code-mirror.po';
+import MachineDeploymentsCreateEditPo from '@/cypress/e2e/po/edit/machine-deployments.po';
+import MachineDeploymentsListPo from '@/cypress/e2e/po/lists/machine-deployments-list.po';
+import ProductNavPo from '@/cypress/e2e/po/side-bars/product-side-nav.po';
+import BurgerMenuPo from '@/cypress/e2e/po/side-bars/burger-side-menu.po';
+
+export default class MachineDeploymentsPagePo extends PagePo {
+  private static createPath(clusterId: string) {
+    return `/c/${ clusterId }/manager/cluster.x-k8s.io.machinedeployment`;
+  }
+
+  static goTo(clusterId: string): Cypress.Chainable<Cypress.AUTWindow> {
+    return super.goTo(MachineDeploymentsPagePo.createPath(clusterId));
+  }
+
+  constructor(private clusterId = 'local') {
+    super(MachineDeploymentsPagePo.createPath(clusterId));
+  }
+
+  static navTo() {
+    const sideNav = new ProductNavPo();
+
+    BurgerMenuPo.burgerMenuNavToMenubyLabel('Cluster Management');
+    sideNav.groups().contains('Advanced').click();
+    sideNav.navToSideMenuEntryByLabel('MachineDeployments');
+  }
+
+  create() {
```

Comment:
the create is part of `ResourceListMastheadPo`. added there it can be accessed via `this.list().masthead()`. applies to other machine-x pages